Add degree-grouping subsection (#22)
greenelab/xswap-manuscript: content/02.body.md

### Degree-grouping

> Our method for degree-preserving permutation gives randomized networks that share few of their edges with the original network.

Reviewer: ```suggestion
Our method for degree-preserving permutation produces randomized networks that share few of their edges with the original network.
```

> Because only degree is preserved, nodes with equal degree can be grouped when summarizing features.

Reviewer: ```suggestion
```

> For example, in link prediction methods, two node pairs with the same source and target degree have link prediction feature values in permuted networks that are drawn from the same distribution.

Reviewer: ```suggestion
The feature values for two node pairs with the same source and target degree are drawn from the same distribution in permuted networks, so nodes with equal degree can be grouped when summarizing features.
```

> Each node pair's feature values in permuted networks can be augmented in this way to have more permuted feature values than permuted networks.

Reviewer: ```suggestion
We used this to augment each node pair's feature values in permuted networks, which allowed these pairs to have more permuted feature values than permuted networks.
```

> For nodes with a common degree, degree grouping can greatly increase the effective number of permutations, even though only a smaller number of permuted networks were generated [@url:https://github.com/greenelab/hetmech/pull/96].

Reviewer: ```suggestion
Degree grouping greatly increased the effective number of permutations for nodes with frequently observed degrees [@url:https://github.com/greenelab/hetmech/pull/96].
```

> We utilized degree grouping throughout our analysis, including in computing the edge prior.

Reviewer: ```suggestion
We used degree grouping throughout our analyses.
```
